src/datachain/query/dataset.py

```diff
@@ -366,12 +367,16 @@ def get_download_callback(suffix: str = "", **kwargs) -> CombinedDownloadCallbac
 
 
 def get_processed_callback() -> Callback:
-    return TqdmCallback({"desc": "Processed", "unit": " rows", "leave": False})
+    return TqdmCallback(
+        {"desc": "Processed", "unit": " rows", "leave": False}, tqdm_cls=tqdm
```

Comment:
Note that `TqdmCallback` (or, `fsspec.Callback` to be specific) can open nested progress bars in certain case. One example is `copy` where one progress bar is about no. of files copied, and the second is the progress bar for individual files being copied.

I am not sure how `notebook_tqdm` works in that case.

Reply:
nested bars work in notebook tqdm (see the demo in the description). Not 100% about nested bars inside copy.

Anyways, we should be better now compared to what it was before. Nesting doesn't work in text mode anyways, the whole experience is broken.